### PR TITLE
Tracing: add zipkin support

### PR DESCRIPTION
**Pull request by MarcelHillmann**
What this PR does / why we need it:
As operator I would like to see how the Prometheus netwerk is working.

To improve this target, I implemented a Zipkin client.

### CODE REVIEW

**Comment by bergquist** [comment]
Hi! Thank you for taking the time to contribute to Grafana. 

Wouldnt you be able to use https://grafana.com/docs/grafana/latest/installation/configuration/#zipkin-propagation to achieve the same results? We are currently swamped with work for 7.0 so response might be slow :)

**Comment by MarcelHillmann** [comment]
Hi @bergquist 

that dosn't works, my backend ist ZIPKIN and not Jaeger.

Thanks
Marcel

File: docs/sources/installation/configuration.md [hunk 1]
```diff
@@ -904,6 +908,59 @@ Default value is `false`.
 
 Setting this to `true` turns off shared RPC spans. Leaving this available is the most common setting when using Zipkin elsewhere in your infrastructure.
 
+### tracing128bit
```
Comment (marefr) [inline docs/sources/installation/configuration.md:911]: Would suggest renaming this to `use_128_bit_trace_ids` and update related code.

**Review by marefr — CHANGES_REQUESTED** [review]
Looks good. Verified it works and that Jaeger still works as before. Need some minor changes. See comments and below

Need to update defaults.ini and sample.ini as well with similar info as you've added to documentation: 
- https://github.com/grafana/grafana/blob/master/conf/defaults.ini
- https://github.com/grafana/grafana/blob/master/conf/sample.ini

**Comment by marefr** [comment]
@MarcelHillmann let us know if you're planning to address review comments and/or you'll need help. Thanks

**Pull request closed without merging**